src/content/en/updates/2019/07/image-support-for-async-clipboard.md

```diff
@@ -85,18 +89,18 @@ document.addEventListener('paste', async (e) => {
 
 ### Security and permissions {: #security-permission }
 
-The `navigator.clipboard` API is only supported for pages served over HTTPS,
+The `navigator.clipboard` property is only supported for pages served over HTTPS,
```

Comment:
Reserve API for whole specs or sets of interfaces. In general you should be specific about whether you talking about a property, method, event, whatever.

Reply:
Gotcha.